src/main/java/com/thealgorithms/conversions/BinaryToOctal.java

```diff
@@ -1,27 +1,16 @@
 package com.thealgorithms.conversions;
 
-import java.util.Scanner;
-
 /**
  * Converts any Binary number to an Octal Number
  *
  * @author Zachary Jones
  */
 public final class BinaryToOctal {
-    private BinaryToOctal() {
-    }
+    private static final int BITS_IN_GROUP = 3;
```

Comment:
The constant `BITS_IN_GROUP ` could be renamed to `BITS_PER_OCTAL_DIGIT` to make its purpose more explicit and improve readability.

Reply:
Yes, this is more readable, thank you.